executor/modes/local/websocket_client.py

```
@@ -321,39 +321,36 @@ def _get_client_ip(self) -> str:
                 return ip
         except Exception:
             logger.debug("Failed to detect IP via hostname resolution", exc_info=True)
 
         try:
             # Method 3: Try to get IP from network interfaces (Linux only)
             result = subprocess.run(
-                ['hostname', '-I'],
-                capture_output=True,
-                text=True,
-                timeout=5
+                ["hostname", "-I"], capture_output=True, text=True, timeout=5
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)